gapic-generator-java/DEVELOPMENT.md

```diff
@@ -2,209 +2,172 @@
 
 ## Set Up
 
-1.  Clone this repo.
+1. Clone this repo.
 
-2.  Copy the Git pre-commit hooks. This will automatically check the build, run
-    tests, and perform linting before each commit. (Symlinks don't seem to work,
-    but if you find a way, please add it here!)
+2. (OPTIONAL) Copy the Git pre-commit hooks. This will automatically check the build, run
+   tests, and perform linting before each commit. (Symlinks don't seem to work,
+   but if you find a way, please add it here!)
 
     ```sh
     cp .githooks/pre-commit .git/hooks/pre-commit
     ```
 
+3. Install [`bazelisk`](https://github.com/bazelbuild/bazelisk) in your `PATH`.
+
 ## Code Formatting
 
--   Run linter checks without actually doing the formatting.
+- Run linter checks without actually doing the formatting.
```

Comment:
They come from the latest https://github.com/googleapis/gapic-generator-java/blob/main/DEVELOPMENT.md (at the root).